readthedocs/templates/projects/notifications/deprecated_config_file_used_email.html

```diff
@@ -13,7 +13,7 @@
     <li><strong>Monday, September 25, 2023</strong>: Fully remove support for building documentation without configuration file v2.</li>
 </ul>
 
-We have identified that the following projects which you maintain are impacted by this deprecation.
+We have identified that the following projects which you maintain, and were built in the last year, are impacted by this deprecation:
```

Comment:
I'd suggest an alternative with more words. If it saves people from reading it 2-3 times, it's actually shorter :innocent: 

```suggestion
We have identified all projects impacted by this deprecation. Out of the projects that you maintain, the following had build activity during the previous 365 days:
```

(same change in txt version)

Reply:
This doesn't seem clearer to me, and outside the scope of this PR. We already went back & forth on this copy in the previous PR?